markdown/3.0/en/ff-suggest.md

```diff
@@ -81,6 +80,49 @@ rather than the rendered: "**Back**packs".
 
 See [Template Engine](documentation/template-engine) for more details.
 
+### Rendered HTML
+
+Container templates provided inside `<ff-suggest>` tag will be rendered inside
+several additional wrapping `<div>` HTML elements.
+
+For example, the following HTML provided by the user:
+
+```html
+<ff-suggest layout="block">
+    <section>
```

Comment:
I think a not empty section would be more informative.

Maybe refer to the `ff-suggest` container from above's section and show an example of its rendered output?

Reply:
I updated HTML with a section with some sample content.